fuel/converters/caltech101_silhouettes.py

```diff
@@ -0,0 +1,64 @@
+import os
+import h5py
+
+from scipy.io import loadmat
+
+from fuel.converters.base import fill_hdf5_file, check_exists
+
+ALL_FILES16 = ['caltech101_silhouettes_16_split1.mat'] #, 'caltech101_silhouettes_16_split1.mat']
+ALL_FILES28 = ['caltech101_silhouettes_28_split1.mat'] #, 'caltech101_silhouettes_28_split1.mat']
+
+
+# TODO: Handel 16x16 dataset
```

Comment:
You can draw inspiration from the [SVHN converter](https://github.com/vdumoulin/fuel/blob/svhn_format_2/fuel/converters/svhn.py) to handle both versions of the dataset with the same converter.